tests/lib/sms/Kconfig

```
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config AT_NOTIF
+	bool "Internal"
+	default y
+	help
+	  Used in tests to enable mocking of AT Command library, i.e., remove
+	  dependency from AT Command Notifications library to AT command library
+
+config SMS_AT_CMD
+	bool "Internal"
```

Why does this have a prompt ?
For this test case I believe this should always be disabled.

```suggestion
	bool
```